consensus/service/src/tx_manager.rs

```diff
@@ -510,4 +836,152 @@ mod tests {
         // The ledger was previously initialized with 3 blocks.
         assert_eq!(block.index, 3);
     }
+
+    #[test_with_logger]
+    #[ignore]
```

Comment:
Why is this ignored?

Reply:
Good catch. I often stub out tests with ignore and unimplemented!, but here I forgot to remove the ignore after it got implemented. Will fix.